test/test_pallas_spmd.py

```diff
@@ -264,12 +302,12 @@ def test_cross_flash_attention_wrapper_segment_ids_spmd(self):
     from jax.experimental.pallas.ops.tpu.flash_attention import flash_attention as jax_flash_attention, SegmentIds
     xs.set_global_mesh(xs.get_1d_mesh("data"))
 
-    q = torch.randn(3, 2, 1024, 4)
-    k = torch.randn(3, 2, 128, 4)
-    v = torch.randn(3, 2, 128, 4)
-    zeros = torch.zeros(3, 32)
-    kv_segment_ids = torch.cat([zeros, zeros + 1, zeros + 2, zeros + 3], dim=1)
-    q_segment_ids = torch.ones(3, q.shape[2], dtype=torch.float32)
+    q = torch.randn(8, 2, 1024, 4)
+    k = torch.randn(8, 2, 128, 4)
+    v = torch.randn(8, 2, 128, 4)
+    zeros = torch.zeros(8, 32)
+    kv_segment_ids = torch.cat([zeros, zeros + 1, zeros + 2, zeros + 4], dim=1)
```

Comment:
I'm not familiar with the kv segment ids. Wondering did you intend to change zero+3 to zero+4?

Reply:
that was unintended indeed. But it shouldn't matter.